master/pkg/model/experiment_config.go

```diff
@@ -193,6 +193,7 @@ type ResourcesConfig struct {
 	NativeParallel bool    `json:"native_parallel"`
 	ShmSize        *int    `json:"shm_size,omitempty"`
 	AgentLabel     string  `json:"agent_label"`
+	ResourcePool   string  `json:"resource_pool"`
```

Comment:
Does this mean users can now specify the resource pool they want to use?

Reply:
yes

Reply:
With the current state of the code, what happens if a user submits an experiment with a `resource_pool` that doesn't exist? What error does the user see?

Reply:
It got an error: `Failed to create experiment: starting experiment: cannot find resource pool: ...`